lib/Exception/XmlLoadException.php

```diff
@@ -30,49 +35,36 @@ public function __construct(string $response, array $errors)
         }
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function __toString(): string
     {
-        return '['.get_class($this).'] '.$this->message."\n".
-            'Response: '."\n".
-            $this->response
-        ;
+        return '[' . static::class . '] ' . $this->message . "\n" .
+            'Response: ' . "\n" .
+            $this->response;
     }
 
-    /**
-     * @param \LibXMLError $error
-     * @param string       $xml
-     *
-     * @return string
-     */
-    private function decodeXmlError(\LibXMLError $error, string $xml): string
+    private function decodeXmlError(LibXMLError $error, string $xml): string
     {
-        $return = $xml[$error->line - 1]."\n";
+        $return = $xml[$error->line - 1] . "\n";
 
         switch ($error->level) {
             case LIBXML_ERR_WARNING:
-                $return .= "Warning $error->code: ";
+                $return .= sprintf('Warning %u: ', $error->code);
```

Comment:
this (and the others below) are really simple string concats, it is not necessary to use sprintf

Reply:
`sprintf` is used as it is more maintainable: easier to read and statically analysable (avoid null-value errors).